.travis.yml

```diff
@@ -34,7 +52,11 @@ install:
   - conda install -q anaconda-client requests filelock contextlib2 jinja2 patchelf ripgrep pyflakes beautifulsoup4 chardet pycrypto glob2 psutil pytz tqdm conda-package-handling py-lief python-libarchive-c
   - pip install pkginfo
   - if [[ "$FLAKE8" == "true" ]]; then
-      conda install -q flake8;
+      if [ `uname -m` = 'aarch64' ]; then
```

Comment:
I suppose equal condition can be checked with '==' not '='.  Modify this.

Reply:
removin this block as discussed, and "=" & "==" both are working fine, anyway updating it to "==" in first architecture check.